citi-onboarding/Aqua-Jr
# Feature/banner and post to blog

# What I did
- Created a database table for the Banner and its route
- Created a database table for the Post to Blog section and its route

# How to test
- Access the branch through your terminal:
```
   git checkout develop
   git pull
   git checkout remote/origin/feature/Banner_and_Post_To_Blog_
```
- Access the server folder: 
```
    cd server
```
- Run either **node index.js** or **nodemon index.js**
- Now, if the previous steps were successful, go to your browser and type 
```    
    localhost:3001/admin
```
- Edit the **Banner** element and access  
```
    localhost:3001/api/banners
```
- Now you should go back to the ADMIN page and repeat the process, only now adding an element to **Post to blog** 
- After this, you should  visit the API page again, but now using this other address:
```
    localhost:3001/api/posttoblog
```
- If you were able to edit both elements and see the changes you made reflected on their respective API pages, that means the code is running as it should.
